scripts/preptest.sh

```diff
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
 #
 # Prep a base Drupal install for memcache testing.
 # This assumes that $WEBDIR contains a working Drupal install.
 
 WEBDIR=/var/www/html
 MEMCACHE_SETTINGS_FILE=memcache.settings.inc
-MEMCACHE_VERSION=7.x-1.2
-DATABASE_NAME=drupal
+MEMCACHE_VERSION=6.x-1.11
```

Comment:
I was planning to have a 6.x version and a 7.x version, to try and keep the script itself as simple as possible. In time the 6.x version will likely become less important/used, but I think this keeps it more likely to work in the far-future should we have to dust it off for any reason.